packages/cli/src/commands/integration/command.ts

```diff
@@ -91,6 +92,77 @@ export const listSubcommand = {
   ],
 } as const;
 
+export const removeSubcommand = {
+  name: 'remove',
+  description:
+    'Unlink and delete resources, and uninstall marketplace integrations',
+  arguments: [
+    {
+      name: 'resource/integration',
+      required: true,
+    },
+    {
+      name: 'project',
+      required: false,
+    },
+  ],
+  options: [
+    {
+      name: 'delete',
+      description: 'limits the resources listed to a designated integration',
+      shorthand: 'D',
+      type: Boolean,
+      deprecated: false,
+      argument: 'NAME',
+    },
+    {
+      name: 'unlink-all',
+      description: 'lists all resources regardless of project',
```

Comment:
Same with this description - why do we mention listing here?

Reply:
Copy-paste miss. Good callout.